src/graphHasCycles.ts

```diff
@@ -13,23 +13,17 @@ export function graphHasCycles(pGraphDependencyMap: Map<string, PGraphNodeWithDe
    */
   const visitMap = new Map<string, boolean>();
 
-  for (const [nodeId, nodes] of pGraphDependencyMap.entries()) {
+  for (const [nodeId] of pGraphDependencyMap.entries()) {
     /**
      * Test whether this node has already been visited or not.
      */
     if (!visitMap.has(nodeId)) {
       /**
        * Test whether the sub-graph of this node has cycles.
        */
-      if (hasCycleDFS(pGraphDependencyMap,  visitMap, nodeId)) {
-        return {
-          hasCycle: true,
-          details: {
-            nodeId,
-            dependsOn: Array.from(nodes.dependsOn),
-            dependedOnBy: Array.from(nodes.dependedOnBy)
-          }
-        };
+      const cycle = hasCycleDFS(pGraphDependencyMap,  visitMap, nodeId);
```

Comment:
the name of the function should be solved, the current name implies that the function returns a boolean